sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -3,14 +3,9 @@
 if (NOT DEFINED UNIFIED_RUNTIME_LIBRARY OR NOT DEFINED UNIFIED_RUNTIME_INCLUDE_DIR)
   include(FetchContent)
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  #commit b38855ed815ffd076bfde5e5e06170ca4f723dc1
-  #Merge: e6343f4 6a2c548
-  #Author: Piotr Balcer <piotr.balcer@intel.com>
-  #Date:   Thu Oct 5 12:15:42 2023 +0200
-  #    Merge pull request #920 from jsji/localcopy
-  #        [UR][L0] Copy prebuilt L0 to avoid leaking shared folder path
-  set(UNIFIED_RUNTIME_TAG b38855ed815ffd076bfde5e5e06170ca4f723dc1)
+  # TODO remove me before merging
+  set(UNIFIED_RUNTIME_REPO "https://github.com/hdelan/unified-runtime.git")
+  set(UNIFIED_RUNTIME_TAG b002e003933169c28f5ac9d31ff8ccda8cb5114a)
```

Comment:
How we should handle this when merging? Is there a new process?

Reply:
I will revert this commit once https://github.com/oneapi-src/unified-runtime/pull/943 passes and then I will ask for this PR to be merged. I think that's the best way to do it although the test will fail for HIP once I revert the commit. So merging will not happen with a green CI run. I'm not sure if that's OK or not?

Reply:
This is actually incorrect. Once the UR PR merges I will modify the CMakeLists.txt in this PR to point to the new checkout SHA in the adapters branch, and hopefully get this merged quickly so that it doesn't conflict with some new patch

Reply:
I have updated this to point to the UR adapters branch instead of the branch in my fork. This should be good to merge once the CI passes